ui/partials/main/thActionButton.html

```diff
@@ -11,18 +11,18 @@
   </button>
   <!-- Menu contents -->
   <ul class="dropdown-menu pull-right">
-    <li title="{{user.loggedin ? '' : 'Must be logged in'}}"
-        class="{{user.loggedin ? '' : 'disabled'}}">
-        <a target="_blank" data-ignore-job-clear-on-click
-           title="Add new jobs to this push"
+    <li><button target="_blank" data-ignore-job-clear-on-click
+           ng-attr-title="{{addNewJobsMenuTitle()}}"
+           ng-class="!user.loggedin || !isDecisionTaskVisible ? 'disabled' : ''"
            href="" class="dropdown-item"
            ng-hide="resultset.isRunnableVisible"
-           ng-click="showRunnableJobs()">Add new jobs</a></li>
-    <li><a target="_blank" data-ignore-job-clear-on-click
-           title="Hide Runnable Jobs"
+           ng-click="showRunnableJobs()">Add new jobs</button></li>
+    <li><button target="_blank" data-ignore-job-clear-on-click
+           ng-attr-title="{{addNewJobsMenuTitle()}}"
+           ng-class="!user.loggedin || !isDecisionTaskVisible ? 'disabled' : ''"
```

Comment:
I'm also going to add an `ng-disabled` here and below L16 so the menu can't be clicked either.

Reply:
Done. I experimented also with applying the `ng-disabled` to the `li` parent, but the disabled circular mouse cursor style blocks reading the beginning of the tooltip. So I've left it this way so the cursor remains arrow and the tooltip readable.